ui/packages/consul-ui/app/components/consul/auth-method/index.scss

```diff
@@ -0,0 +1,12 @@
+.consul-auth-method-list ul {
+  .detail span::before {
+    margin-right: 6px;
+  }
+  .type {
+    @extend %pill-200, %frame-gray-600;
+  }
+  .locality::before {
+    @extend %with-public-default-mask, %as-pseudo;
+    margin-right: 6px;
```

Comment:
Is it possible to use the `dl`/`%icon-defintion` pattern here? Or at least use the same margin from there (I think its 4px not 6px)

Reply:
It doesn't make sense to make this into a `dl`. I'll update the margin.